packages/experimental/ExperimentalIcon/src/SvgIcon/SvgIcon.types.ts

```diff
@@ -5,11 +5,32 @@ import { IViewProps } from '@fluentui-react-native/adapters';
 export const svgIconName = 'SvgIcon';
 
 export interface SvgIconProps extends IViewProps {
+  /**
+   * Icon color.
+   */
   color?: ColorValue;
+  /**
+   * Icon height.
+   */
   height?: number;
+  /**
+   * Path to a local icon.
+   */
   src?: React.FC<SvgProps>;
+  /**
+   * Style object which contains ImageStyle props.
+   */
   style?: StyleProp<ImageStyle>;
+  /**
+   * URI of a remote icon.
+   */
   uri?: string;
+  /**
+   * Viewbox.
```

Comment:
This isn't a very useful comment, is there more description we can add here? Maybe how the prop should be formatted, or something?